server/src/main/java/com/linecorp/centraldogma/server/CentralDogma.java

```diff
@@ -879,6 +880,16 @@ private void configureMetrics(ServerBuilder sb, MeterRegistry registry) {
         ExecutorServiceMetrics.monitor(registry, ForkJoinPool.commonPool(), "commonPool");
     }
 
+    private static MeterIdPrefixFunction repoAwareMeterIdPrefixFunction() {
+        final MeterIdPrefixFunction meterIdPrefix = MeterIdPrefixFunction.ofDefault("api");
+        return meterIdPrefix.andThen((registry0, log, meterIdPrefix0) -> {
+            final ServiceRequestContext ctx = (ServiceRequestContext) log.context();
+            final String project = firstNonNull(ctx.pathParam("projectName"), "none");
+            final String repo = firstNonNull(ctx.pathParam("repoName"), "none");
```

Comment:
Central Dogma conventionally uses `projectName` and `repoName` for path params.
I did a full scan and there were no exceptions.